docs/glossary.mdx

```diff
@@ -1,294 +1,360 @@
+### best tip
 
-### Block
+The blockchain's latest block with the highest [chain strength](#chain-strength) known to the daemon.
 
-A set of transactions and consensus information that extend the state of the network. Includes a proof that the current state of the network is fully valid.
+### block
 
-### Blockchain
+A set of transactions and consensus information that extend the state of the network, a block includes a proof that the current state of the network is fully valid.
+
+### blockchain
 
 The data structure that is used in a cryptocurrency to maintain a shared state of all accounts in the network.
 
-### Block Confirmations
+### block confirmations
 
-The number of blocks added after the reference block. As the number of confirmations increases, the likelihood of a reorganization decreases, thereby increasing the likelihood of all transactions in the reference block being confirmed.
+The number of blocks added after the reference block. As the number of confirmations increases, the likelihood of a [reorganization](/glossary#reorganization) decreases, thereby increasing the likelihood of all transactions in the reference block being confirmed.
 
-### Block Fill Rate
+### block fill rate
 
-The proportion of [slots](#slot) that should contain a block. Some slots are intentionally empty to ensure the network can "catch up" in case of delay of messages.
+The proportion of [slots](#slot) that should contain a block. Some slots are intentionally empty to ensure the network can [catch up](/glossary#catchup) in case of delay of messages.
 
-### Block Producer
+### block producer
 
 A node that participates in a process to determine what blocks it is allowed to produce and then produces blocks containing transactions that can be broadcast to the network.
 
-### Bootstrap
+### bootstrap
+
+Part of the [syncing](#syncing) process of a node, bootstrapping gets the current [root](#root-of-transition-frontier) of the [transition frontier](#transition-frontier) from peers. Additional [transitions](#transition) obtained during the [catchup](#catchup) process are applied from this initial root state.
 
-Bootstrapping is part of the [syncing](#syncing) process of a node. Bootstrapping gets the current [root](#root-of-transition-frontier) of the [transition frontier](#transition-frontier) from peers. Additional [transitions](#transition) obtained during the [catchup](#catchup) process are applied from this initial root state.
+### breadcrumb
 
-### Breadcrumb
+A node in the [transition frontier](#transition-frontier) that contains the external transition, staged ledger, and pending coinbases and is generated by applying the transition to the prior state.
 
-A breadcrumb is a "node" in the [transition frontier](#transition-frontier). A breadcrumb contains the external transition, staged ledger, and pending coinbases and is generated by applying the transition to the prior state.
+## C
 
-### Catchup
+### catchup
```

Comment:
Is there a product reason that "catchup" is one word for Mina? 
Correct spelling = catch up